### PR TITLE
+bench Add pekko grpc bench

### PR DESCRIPTION
**Pull request by He-Pin**
Pekko is a fork of Akka, and now Add the pekko bench and let's see future improvement of both.

### CODE REVIEW

**Comment by He-Pin** [comment]
```
Response time histogram:

Latency distribution:

Status code distribution:
  [Unavailable]   1878499 responses   
  [Canceled]      174 responses       

Error distribution:
  [906]     rpc error: code = Unavailable desc = write tcp 127.0.0.1:59770->127.0.0.1:50051: write: broken pipe   
  [6344]    rpc error: code = Unavailable desc = write tcp 127.0.0.1:59584->127.0.0.1:50051: write: broken pipe   
  [3764]    rpc error: code = Unavailable desc = write tcp 127.0.0.1:59640->127.0.0.1:50051: write: broken pipe   
  [14777]   rpc error: code = Unavailable desc = write tcp 127.0.0.1:46334->127.0.0.1:50051: write: broken pipe   
  [1045]    rpc error: code = Unavailable desc = write tcp 127.0.0.1:597[86](https://github.com/LesnyRumcajs/grpc_bench/actions/runs/6112755875/job/16590903181?pr=381#step:4:87)->127.0.0.1:50051: write: broken pipe   
  [46[94](https://github.com/LesnyRumcajs/grpc_bench/actions/runs/6112755875/job/16590903181?pr=381#step:4:95)]    rpc error: code = Unavailable desc = write tcp 127.0.0.1:59418->127.0.0.1:50051: write: broken pipe
```

Have no idea what's going on
https://github.com/LesnyRumcajs/grpc_bench/actions/runs/6112755875/job/16590903181?pr=381#step:4:427

**Comment by He-Pin** [comment]
@gcnyin I'm on windows, would you like to help out , thanks.

**Comment by He-Pin** [comment]
Ok, it no works.

**Comment by He-Pin** [comment]
@LesnyRumcajs Would you like to take a review on this, thanks.

File: scala_pekko_bench/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+FROM hseeberger/scala-sbt:11.0.7_1.3.13_2.11.12 as BUILDER
```
Comment (gcnyin) [inline scala_pekko_bench/Dockerfile:1]: I think https://hub.docker.com/r/sbtscala/scala-sbt/tags is better choice.

Comment (He-Pin) [reply scala_pekko_bench/Dockerfile:1]: Thanks, what about after this got merged and you submitted a new pr for it？

File: scala_pekko_bench/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+WORKDIR /app
+
+# Make sure docker can cache a few static setup things
+
+# initialize only sbt
+RUN mkdir -p /app/project
+COPY scala_pekko_bench/project/build.properties /app/project
+RUN sbt exit
```
[the remark this answers is not in the document]
Comment (He-Pin) [reply scala_pekko_bench/Dockerfile:10]: Copied from akka grpc one, but this can be optimized.

**Comment by He-Pin** [comment]
@LesnyRumcajs I have updated this PR

**Review by LesnyRumcajs — APPROVED** [review]
LGTM, thanks.

**Pull request merged**